packages/flutter_tools/lib/src/ios/devices.dart

```diff
@@ -382,6 +382,7 @@ class IOSDevice extends Device {
       if (debuggingOptions.traceAllowlist != null) '--trace-allowlist="${debuggingOptions.traceAllowlist}"',
       if (debuggingOptions.endlessTraceBuffer) '--endless-trace-buffer',
       if (debuggingOptions.dumpSkpOnShaderCompilation) '--dump-skp-on-shader-compilation',
+      if (debuggingOptions.purgePersistentCache) '--purge-persistent-cache',
       if (debuggingOptions.verboseSystemLogs) '--verbose-logging',
```

Comment:
there is also an iOS unit test for flag passing, please update

Reply:
Done.